demo/src/main/java/com/example/demo/DemoApplicationForSQ.java

```diff
@@ -1,16 +1,14 @@
 package com.example.demo;
 
-import org.springframework.boot.SpringApplication;
-import org.springframework.boot.autoconfigure.SpringBootApplication;
-
-@SpringBootApplication
 public class DemoApplicationForSQ {
 
-	public static void main(String[] args) {
-		SpringApplication.run(DemoApplicationForSQ.class, args);
+	public void doSomething() {
 	}
 
-	public void doSomething() {
+	public void doSomething2() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1186)